docs/src/explicit/QPRK.md

```diff
@@ -0,0 +1,14 @@
+# OrdinaryDiffEqQPRK
+
+This article is a stub.
```

Comment:
Q is quadruple?

Reply:
Quad Precision, it's specialized RK methods for quad-precision arithmetic, so values in the Float128 range.